Detectors/ITSMFT/ITS/simulation/src/Detector.cxx

```diff
@@ -1220,83 +1220,4 @@ Hit* Detector::addHit(int trackID, int detID, const TVector3& startPos, const TV
   return &(mHits->back());
 }
 
-void Detector::Print(std::ostream* os) const
```

Comment:
It's ok to suppress these methods, but I think you did not remove them from the header

Reply:
done